src/05-led-roulette/README.md

```diff
@@ -2,9 +2,8 @@
 
 Alright, let's start by building the following application:
 
-# TODO REPLACE THIS
 <p align="center">
-<img src="https://i.imgur.com/0k1r2Lc.gif">
+<img src="https://imgur.com/download/RWBWYX4/">
```

Comment:
Could we host the video directly on this repository? Here the mp4 version: https://i.imgur.com/RWBWYX4.mp4